frontend/src/components/Menu.js

```diff
@@ -1,41 +1,41 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
-import MenuComponent from '../components/MenuComponent'
+import { NavLink } from 'react-router-dom'
+import { People, Newspaper, Shop, ArrowBarLeft } from 'react-bootstrap-icons'
 
-const menuItems = [
-  {
-    path: '/super-admin/users',
-    title: 'users',
-  },
-  {
-    path: '/super-admin/products',
-    title: 'products',
-  },
-  {
-    path: '/super-admin/events',
-    title: 'events',
-  },
-  {
-    path: 'logout',
-    title: 'menu.signout',
-  },
-]
-
-export default function Menu() {
+const Menu = () => {
   const { t } = useTranslation()
+
   return (
-    <div className="d-flex" id="wrapper">
-      <div className="bg-light border-right" id="sidebar-wrapper">
-        <div className="list-group list-group-flush">
-          {menuItems.map((item, index) => (
-            <MenuComponent
-              key={index}
-              title={t(item.title)}
-              path={item.path}
-            ></MenuComponent>
-          ))}
-        </div>
+    <div className="border-right">
+      <div className="list-group">
+        <NavLink className="nav-link" to="/super-admin/users">
+          <h4>
+            <People /> {t('menu.users')}
```

Comment:
Ezt vissza lehetne írni, gondolom az ikonok miatt írtad át. De a prop-ban lehet átadni komponenst is: `<MenuComponent Icon={People}/>`
Belül meg: 
`<Icon />`
A fenti listában pedig szintén lehet komponenseket is rakni 
```
{
title: 'menu.signout',
icon: ArrowBarLeft,
}
```

Reply:
Igen, próbáltam a listához hozzáaadni, és propsként mapelésnél leadni komponensként az ikonokat, de a helyes syntaxist nem találtam meg.